HACKING.md

```diff
@@ -219,6 +236,141 @@ or
 
 The images will then be built and stored in the cluster VM's local image store and then pushed to your configured Docker registry. 
 
+#### Local build with push to Docker registry used by Kind
+
+When developing locally you might want to push the docker images to the docker repository running as container in your Docker daemon and used by your Kind cluster deployed in the same Docker daemon. This can be quicker and more convenient than pushing to Docker Hub and works even without a network connection.
```

Comment:
```suggestion
When developing locally you might want to push the docker images to the docker repository running as a container in your Docker daemon and used by your Kind cluster deployed in the same Docker daemon. This can be quicker and more convenient than pushing to Docker Hub and works even without a network connection.
```